app/src/main/java/com/onelist/ui/MainActivity.kt

```diff
@@ -61,19 +60,19 @@ class MainActivity : ComponentActivity() {
             OutlinedTextField(
                 value = itemName,
                 onValueChange = { itemName = it },
-                label = { Text(stringResource(R.string.ItemName)) },
+                label = { Text(stringResource(R.string.item_name)) },
```

Comment:
Keeping the names in string.xml consistent. All keys are lower snake case instead of having some Upper Camel Case.